### PR TITLE
feat: add placeholderbrand page

### LINKED ISSUE #974: Integrate `/placeholderbrand` page
### Describe the feature

This page consists of many previously developed components, we now need to integrate them on one page. We will replace the example with real informations ones later on. Please take a look at our Figma, the trailing contact section can be ignored for now.

Image

### Why is this feature needed?

The `/placeholderbrand` a new page our `landing-page` to communicate our one of our new projects. 

### **Until publicly announced, we will not disclose any informations about this project and are using example data.**

### PR DESCRIPTION
**Pull request by DasProffi**
## Which issues does this pull request close?
closes #974

## [OPTIONAL] Give testing instructions to reviewers
Open the `/placeholderbrand` page

### CODE REVIEW

**Comment by DasProffi** [comment]
- [ ] Personal Contacts with Profiles have to be added

File: landing-page/pages/placeholderbrand/index.tsx [hunk 1]
```diff
@@ -0,0 +1,314 @@
...
+const PlaceholderbrandFAQSection = () => {
+  const translation = useTranslation(defaultPlaceholderbrandFAQTranslation)
+  return (
+    <div className={tw('flex flex-col w-full max-w-[1000px]')}>
+      <Span type="title" className={tw('text-hw-secondary-400 !text-3xl')}>{translation.title}</Span>
+      <Span>{translation.subTitle}</Span>
+      <div className={tw('flex flex-col gap-y-4 mt-4')}>
+        <FAQSection
+          entries={[
+            /* TODO fectch entries */
```
Comment (MaxSchaefer) [inline landing-page/pages/placeholderbrand/index.tsx:259]:
```suggestion
```
No fetch needed. We will hardcode the FAQ.

**Review by MaxSchaefer — CHANGES_REQUESTED** [review]
Bildschirmfoto 2024-04-09 um 11 12 42 PM

Please check out the spacing of the headers

**Comment by DasProffi** [comment]
> Please check out the spacing of the headers

What spacing are you refering to?
- the horizontal padding
- the title and subtitle (which start at the same horizontal position)

**Comment by MaxSchaefer** [comment]
> > Please check out the spacing of the headers
> 
> What spacing are you refering to?
> 
> * the horizontal padding
> * the title and subtitle (which start at the same horizontal position)

The 

> > Please check out the spacing of the headers
> 
> What spacing are you refering to?
> 
> * the horizontal padding
> * the title and subtitle (which start at the same horizontal position)

@DasProffi spacing between title-and-subtitle and content (like profiles and faqs)

**Review by MaxSchaefer — APPROVED** [review]

**Pull request merged**